src/client/sync/mod.rs

```diff
@@ -3,26 +3,46 @@
 
 //! Synchronous Modbus client
 
-// TODO: Add missing documentation
 #![allow(missing_docs)]
 
-/// RTU client connections
-#[cfg(feature = "rtu")]
+#[cfg(feature = "rtu-sync")]
 pub mod rtu;
 
-/// TCP client connections
-#[cfg(feature = "tcp")]
+#[cfg(feature = "tcp-sync")]
 pub mod tcp;
 
+use std::{future::Future, time::Duration};
+
+use futures::future::Either;
+
+use crate::{frame::*, slave::*};
+
 use super::{
     Client as AsyncClient, Context as AsyncContext, Reader as AsyncReader, SlaveContext,
     Writer as AsyncWriter,
 };
 
-use crate::frame::*;
-use crate::slave::*;
-
-use std::io::Result;
+/// Re-export
+pub use std::io::Result;
```

Comment:
Why do we want to re-export this?

Reply:
Removed: 99b2ec952d1cf76432798ed0678607d88c6bde5d

The submodules were previously importing this from `super`, which is confusing and uncommon for a private import. Now `std::io::Result` is imported directly.